scripts/thaliPrerequisite.js

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+/*
+ This script will do the following tasks
+ 0. Clone the working jxcore-cordova plugin
+ 1. Fix issue on "can not replace existing file"
+ 2. Install the thali node module to global (until we get thali in npm repo)
+ 3. Move the build-extras.gradle to platform
+ 4: Create a dummy file to prevent future execution by other plugin add operation
+ */
+
+
+var fs = require('fs');
+var exec = require('child_process').execSync;
+
+
+var rootdir = process.argv[2];
+//Dummy file to prevent the execution of this script when the developer add other Cordova plugins
+var dummyFile = rootdir + '/../plugins/org.thaliproject.p2p/dummy.js';
+
+//Replace JXcoreExtension.java
```

Comment:
Please note that the lines I asked to be removed are still in the updated file, thaliPostInstall.js.